src/kubernetes/installer.cr

```diff
@@ -388,4 +389,33 @@ class Kubernetes::Installer
       first_master.private_ip_address
     end
   end
+
+  private def etcd_backup_settings
+    if !settings.datastore.etcd.backups.enabled
+      return "--etcd-disable-snapshots"
+    end
+
+    opts = [] of String
+
+    backups = settings.datastore.etcd.backups
+    opts << "--etcd-snapshot-retention=#{backups.retention}" if backups.retention != nil
```

Comment:
Why not use the `present?` helper here too?

Reply:
I didn't use the `present?` helper since it takes in a `String | Nil`, but retention is a `Int32`

Reply:
Gotcha!